app/Http/Controllers/Admin/OrderController.php

```
@@ -99,11 +102,24 @@ public function update(Request $request, Order $order)
+        $this->orderService->voidOrder($deviceOrder);
         $deviceOrder->update(['status' => OrderStatus::VOIDED]);
-        $deviceOrder->delete();
+        return redirect()->back()->with('success');
+    }
+
+    public function complete(Request $request) {
+
+        $orderId = $request->input('order_id');
+
+        // Run the console command
+        $exitCode = Artisan::call('broadcast:order-completed', [
```

PHPMD (2.15.0)

Avoid unused local variables such as '$exitCode'. (undefined)

(UnusedLocalVariable)